### DISCUSSION TITLE
trying to report a bug (?) issue in VScode using their Report Issue

### DISCUSSION THREAD

**Discussion by JoHenri** [community/community#116522] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Not at all sure this is the right forum, but I tried to report an issue with Visual Studio Code to Microsoft and they directed me to GitHub.  I tried to use the VS Code Help->Report Issue and got an error.  I'm trying to use the old code layout where I've got a main() rather than the new way.  I was told I could use
dotnet new console --use-program-main
but when I type that I get an error saying that is not recognized.
I am using Visual Studio Code 1.87.2 on a Dell Precision 3530 laptop running Win 10 pro with recent updates. Here’s what I get

![image](https://github.com/community/community/assets/92395718/bceaf965-262f-45e5-811e-1c1c4a2ac6ef)

**Comment by anouar-garbaa** [comment]
hello! i think that  that the --use-program-main option wasn't available in the .NET SDK version you were using when you tried the command.
so, try to update the .NET SDK:
* first, Open a terminal window (Command Prompt or PowerShell).
then, type  dotnet --info and press Enter. This will display your current .NET SDK version.
* update the .NET SDK. Download the latest version from the official Microsoft website 
(https://dotnet.microsoft.com/en-us/download).
* Open a new terminal window to ensure the updated version is loaded.
Use the command dotnet new console --use-program-main again. 
This should now create a new console project with the traditional Main method layout.

i hape that help to solve the problem !!
